msg/InternalCombustionEngineControl.msg

```diff
@@ -0,0 +1,8 @@
+uint64 timestamp        		# time since system start (microseconds)
+
+bool ignition_on          		# activate/deactivate ignition
+float32 throttle_control		# [0,1] - Motor should idle with 0. Includes slew rate if enabled.
+float32 choke_control			# [0,1]
+float32 starter_engine_control		# [0,1]
+
+uint8 user_request			# user intent for on/off
```

Comment:
Anything you had in your table in the docs should be captured here in comments

```
| `ignition_on` | True/False | Activate/deactivate |
| `throttle_control` | [0, 1] | Motor should idle with 0. Includes slew rate if enabled. |
| `choke_control` | [0, 1] | 1 closes the air inlet |
| `starter_engine_control` | [0, 1] | 1 if engine start should be delayed |
| `user_request` | On/Off | User intent |
```

Reply:
Good point, let's extend it. Though only anything *ICEControl.msg* related. The "user_request" and "1 if engine start should be delayed" are not part of that.